gateway-js/src/__tests__/integration/networkRequests.test.ts

```diff
@@ -499,9 +507,9 @@ describe('Downstream service health checks', () => {
 
       // Update
       mockSupergraphSdlRequestSuccessIfAfter(
-          'originalId-1234',
-          'updatedId-5678',
-          getTestingSupergraphSdl(fixturesWithUpdate),
+        'originalId-1234',
+        'updatedId-5678',
+        getTestingSupergraphSdl(fixturesWithUpdate as any),
```

Comment:
Seems odd this was cast to `as any`. I have a related patch on my current branch that should resolve this, though I'm not sure why there are TS errors that didn't exist before.

```ts
// federation-integration-testsuite-js/src/fixtures/index.ts
export interface Fixture {
  name: string;
  url: string;
  typeDefs: DocumentNode;
  resolvers?: GraphQLResolverMap<any>
}

const fixtures: Fixture[] = [
  accounts,
  books,
  documents,
  inventory,
  product,
  reviews,
];

const fixturesWithUpdate: Fixture[] = [
  accounts,
  books,
  documents,
  inventory,
  product,
  reviewsWithUpdate,
];

const fixturesWithoutTag: Fixture[] = [
  accountsWithoutTag,
  books,
  documents,
  inventory,
  product,
  reviewsWithoutTag,
];
```